src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx

```diff
@@ -251,54 +253,68 @@ If the currently selected repository isn't associated with a service on New Reli
 
 <img
-  title="The broadcast icon for a service in the Observability section."
-  alt="A screenshot of the broadcast icon for a service in the Observability section."
-  src={codestreamCodeLevelMetricsBroadcastIcon}
+  title="Code-level metrics section showing poorly performing methods."
+  alt="A screenshot of the code-level metrics section showing poorly performing methods."
+  src={codestreamCodeLevelAnomalies}
 />
 
+CodeStream identifies these methods by looking for anomalies in either error rate or average duration. If you’re using New Relic’s [change tracking](/docs/change-tracking/change-tracking-introduction/) to send deployment information, anomalies are calculated by comparing the performance since your last release against the preceding three weeks. Otherwise we use the last seven days. You can control these timeframes, as well as the thresholds for determining whether or not a given method is performing poorly, by hovering over the “Code-Level Metrics” heading and clicking on the gear icon. 
+
+In addition to the Code-Level Metrics section, CodeStream will also let you know about poorly performing methods via [desktop notifications](/docs/codestream/codestream-integrations/notifications#additionaldesktop) in either VS Code or JetBrains IDEs.
+
 ### Requirements [#clm-requirements]
 
+In-editor CodeLenses are available for all of New Relic’s APM languages. Anomaly detection, however, is currently only available for Java, .NET, Python and Ruby.
+
+To see metrics, your service must meet the requirements listed below, and the service should have collected data in the last 30 minutes. The New Relic agent attempts to automatically collect data for functions on classes that are tied to HTTP requests. In many cases, and with many frameworks that use an MVC framework, these are often methods on a Controller class. 
+
+Because all requests by the New Relic agent are not collected all the time, it’s possible that low trafficked methods will not see any data. If data is missing for a specific method that you wish to see results for, custom instrumentation can be used to fill any gaps. See guidance for [Java](/docs/apm/agents/java-agent/custom-instrumentation/java-custom-instrumentation), [.NET](/docs/apm/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation), [PHP](/docs/apm/agents/php-agent/features/php-custom-instrumentation), [Python](/docs/apm/agents/python-agent/custom-instrumentation/python-custom-instrumentation), [Ruby](/docs/apm/agents/ruby-agent/api-guides/ruby-custom-instrumentation), [Go](/docs/apm/agents/go-agent/instrumentation/instrument-go-transactions) and [Node.js](/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation). Also note that for Node.js, code-level metrics are not captured for anonymous functions.
```

Comment:
```suggestion
Because all requests by the New Relic agent are not collected all the time, it’s possible that low-traffic methods will not see any data. If data is missing for a specific method that you wish to see results for, you can use custom instrumentation to fill any gaps. See guidance for [Java](/docs/apm/agents/java-agent/custom-instrumentation/java-custom-instrumentation), [.NET](/docs/apm/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation), [PHP](/docs/apm/agents/php-agent/features/php-custom-instrumentation), [Python](/docs/apm/agents/python-agent/custom-instrumentation/python-custom-instrumentation), [Ruby](/docs/apm/agents/ruby-agent/api-guides/ruby-custom-instrumentation), [Go](/docs/apm/agents/go-agent/instrumentation/instrument-go-transactions) and [Node.js](/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation). Also note that for Node.js, code-level metrics are not captured for anonymous functions.
```